doc/TextMate.md

```diff
@@ -0,0 +1,96 @@
+# TextMate Support in Theia
+
+TextMate grammars allow us to colorize most source files with accurate
+decorations, even if it is only at the syntactic level (no language deep
+understanding). Semantic coloring can be provided by Language Servers, among
```

Comment:
I don't think semantic coloring is part of the LSP (yet).

Reply:
No but I felt like it was worth mentioning, to give a bit of context.

Reply:
Theia supports a provisional extension. So I think it is fair to mention it like that.